src/LiquidityPool.sol

```diff
@@ -62,7 +62,7 @@ contract LiquidityPool is Initializable, OwnableUpgradeable, UUPSUpgradeable, IL
     address public etherFiAdminContract;
     bool public DEPRECATED_whitelistEnabled;
     mapping(address => bool) public DEPRECATED_whitelisted;
-    mapping(address => BnftHoldersIndex) public bnftHoldersIndexes;
+    mapping(address => BnftHoldersIndex) public validatorSpawner;
```

Comment:
what does validatorSpawner mean

Reply:
the addresses allowed to submit the validators keys. these are syko, jv's gnosis safes, e.g.,